.bumpversion.cfg

```diff
@@ -12,13 +12,6 @@ serialize =
 
 [bumpversion:file:scrapy/VERSION]
 
-[bumpversion:part:prerel]
-optional_value = gamma
-values = 
-	dev
-	rc
-	gamma
-
 [bumpversion:part:prerelversion]
```

Comment:
@eliasdorneles , we could get rid of `{prerelversion}` and `[bumpversion:part:prerelversion]` section too, I suppose.